src/ImageSharp/Quantizers/Wu/WuQuantizer.cs

```diff
@@ -118,13 +103,13 @@ public sealed class WuQuantizer<TColor> : IQuantizer<TColor>
         /// </summary>
         public WuQuantizer()
         {
-            this.vwt = LongPool.Rent(TableLength);
-            this.vmr = LongPool.Rent(TableLength);
-            this.vmg = LongPool.Rent(TableLength);
-            this.vmb = LongPool.Rent(TableLength);
-            this.vma = LongPool.Rent(TableLength);
-            this.m2 = FloatPool.Rent(TableLength);
-            this.tag = BytePool.Rent(TableLength);
+            this.vwt = WuArrayPool.LongPool.Rent(TableLength);
```

Comment:
I think we should aim to replace these individual pools with `PinnedBuffer`-s when touching different implementations along the library to centralize the buffer logic, and make future refactor work easier. 
(It won't be pinned I'm almost done removing all the pointers.)